lib/tasks/annotations.rake

```diff
@@ -0,0 +1,23 @@
+namespace :annotations do
+  desc 'Remediate annotations'
+  task remediate: :environment do
+    ActiveRecord::Base.transaction do
+      puts "Updating #{Annotot::Annotation.count} annotations"
+      Annotot::Annotation.find_each do |annotation|
+        print '.'
+        # Remove "@context"
+        data = JSON.parse(annotation.data)
+        data.delete('@context')
```

Comment:
Should we re-add the IIIF `@context` that newer annotations have? Or are we committed to removing them everywhere?

Reply:
Yeah I was looking into that, but I couldn't easily find a context that is being used by Mirador, so just opted to leave it blanks /shrug. What context would you expect?

Reply:
The new annotations have:

```
"@context"=>"http://iiif.io/api/presentation/2/context.json"
```

Which seems weird, but 🤷‍♂️

Reply:
I saw the `AnnotationList` context as that here: http://iiif.io/api/presentation/2.1/#annotation-list

But I didn't see a context within the specific resource.. that is why i kind of punted on it here.

Reply:
🆒